install-sbo.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+set -e
+set -u
+REGISTRY_PASSWORD?=""
+
+#The SBO can also be installed in a vanilla kubernetes cluster. A prerequisite for this would be to add credentials for the registry.redhat.io in the cluster. Steps to be followed to achieve the same are listed below:
+#1) Follow [Red Hat Container Registry Authentication Steps](https://access.redhat.com/RegistryAuthentication)
+#2) Verify that your credentials are correct using docker login -u <your_username> -p <your_passwd> registry.redhat.io
+echo "${REGISTRY_PASSWORD}" | docker login -u "$USERNAME" --password-stdin registry.redhat.io
```

Comment:
placing a password in env is a security issue... instead, we can use just `docker login registry.redhat.io`. If user did not login before, he/she would be prompted for entering username/password - otherwise, docker would try to use the existing credentials stored in `~/.docker/config.json` and exit with code 0 if everything is ok. We should check the exit code and fail if not zero.

Reply:
Done